r/tests/testthat/test-dplyr.R

```diff
@@ -593,3 +594,12 @@ test_that("bad explicit type conversions with as.*()", {
   )
 
 })
+
+test_that("No duplicate field names are allowed in an arrow_dplyr_query", {
+  expect_error(
+    Table$create(tbl, tbl) %>%
+      filter(int > 0),
+    regexp = "The following field names were found more than once in the data: int, dbl, dbl2, lgl, false, chr, fct, verses, padded_strings"
```

Comment:
The last change altered the error message slightly so you'll need to fix (I'd make a `suggestion` here but you'll still need to pull and run to confirm)